### PR TITLE
Apply missing patches to v1.6.4-rhel branch

### PR DESCRIPTION
**Pull request by mheon**
A patch was missed when we initially created the branch. Also, apply the fix for CVE-2020-14370.

### CODE REVIEW

**Comment by mheon** [comment]
@baude @jnovy PTAL

This may not pass CI initially. If it does not, I will grab the patches we threw on the other v1.6 branch that fixed it there.

**Comment by jnovy** [comment]
@mheon let's wait with CVE-2020-14370 as it was not approved for 2.0-8.3.0 yet (it's Moderate).

**Comment by jnovy** [comment]
@mheon could you please put reference to https://bugzilla.redhat.com/show_bug.cgi?id=1875289 in the signal 23 proxy commit?

**Comment by mheon** [comment]
Done. I added a commit to fix the CI config for the branch.

**Comment by mheon** [comment]
@cevich Poke - mind taking a look at this Cirrus error?

**Comment by rhatdan** [comment]
LGTM
What is the META failure?

**Comment by mheon** [comment]
Dunno, need @cevich to look at it

File: .cirrus.yml [hunk 1]
```diff
@@ -92,7 +92,7 @@ gating_task:
 
     # Runs within Cirrus's "community cluster"
     container:
-        image: "quay.io/libpod/gate:latest"
+        image: "quay.io/libpod/gate:v1.6"
```
Comment (cevich) [inline .cirrus.yml:95]: good catch!

File: .cirrus.yml [hunk 2]
```diff
@@ -315,329 +187,6 @@ image_prune_task:
     script: '/usr/local/bin/entrypoint.sh |& ${TIMESTAMP}'
```
Comment (cevich) [inline .cirrus.yml:188]: Please nuke image_prune as well.  It only ever runs on the master branch...and is currently disabled anyway.

File: .cirrus.yml [hunk 3]
```diff
@@ -648,22 +197,8 @@ success_task:
     # ignores any dependent task conditions
     depends_on:
         - "gating"
-        - "vendor"
-        - "varlink_api"
-        - "build_each_commit"
-        - "build_without_cgo"
-        - "meta"
         - "image_prune"
```
Comment (cevich) [inline .cirrus.yml:200]: and kill image_prune here as well.

**Comment by mheon** [comment]
@cevich CI still failing over here - Cirrus is complaining about being unable to decrypt a value?

**Pull request merged**